prow/spyglass/lenses/buildlog/buildlog.ts

```diff
@@ -46,16 +46,45 @@ function ansiToHTML(orig: string): string {
   });
 }
 
-async function replaceElementWithContent(element: HTMLDivElement) {
+interface ArtifactRequest {
+  artifact: string;
+  bottom: number;
+  length: number;
+  offset: number;
+  startLine: number;
+  top: number;
+}
+
+async function replaceElementWithContent(element: HTMLDivElement, top: number, bottom: number) {
+  // +length => make it a number (as opposed to a string)
```

Comment:
Oh, cool! :D

However,
Style: Looks like this should be Number()? https://google.github.io/styleguide/tsguide.html#type-coercion

Reply:
Done